### ISSUE TITLE
Images not loading in bundle builds

### ISSUE THREAD

**Issue by thebreiflabb** [ReactWindows/react-native-windows#925]
After upgrading react-native-windows from v0.37.1 to v0.38.0 (also relevant for v0.39.0-rc.0) "require" image sources does not work when running bundle builds (DebugBundle/ReleaseBundle).

E.g. when using this syntax for the image source:
`<Image source={require('app/resources/images/cm-logo-white.png')} />`

The issue seems to boil down to `SetUriFromSingleSource` in [ReactImageManager.cs](https://github.com/ReactWindows/react-native-windows/blob/v0.38.0/ReactWindows/ReactNative/Views/Image/ReactImageManager.cs#L310) being changed in v0.38.0
It looks like `ImageCache.Instance.GetFromCacheAsync` cannot handle URI's with the `ms-appx` protocol
![image-exception](https://cloud.githubusercontent.com/assets/16031158/20926675/f23d0dc0-bbbd-11e6-8934-1e9f99c8a8b9.png)

I'm not that proficient in C# so not saying this is a proper solution, especially since I would think there is a reason behind the ImageCache, but the images are loaded properly if I change this code (from the screenshot):

```csharp
var image = await ImageCache.Instance.GetFromCacheAsync(new Uri(source), true);
var metadata = new ImageMetadata(source, image.PixelWidth, image.PixelHeight);
OnImageStatusUpdate(view, ImageLoadStatus.OnLoad, metadata);
imageBrush.ImageSource = image;
OnImageStatusUpdate(view, ImageLoadStatus.OnLoadEnd, metadata);
```

To this:
```csharp
var image = new BitmapImage();

using (var stream = await BitmapImageHelpers.GetStreamAsync(source))
{
    await image.SetSourceAsync(stream);
}

var metadata = new ImageMetadata(source, image.PixelWidth, image.PixelHeight);
OnImageStatusUpdate(view, ImageLoadStatus.OnLoad, metadata);
imageBrush.ImageSource = image;
OnImageStatusUpdate(view, ImageLoadStatus.OnLoadEnd, metadata);
```

**Comment by rozele** [comment]
Thanks @thebreiflabb I'll get this fixed.

**Comment by rozele** [comment]
@thebreiflabb I've published 0.38.1 with this fix.  Thanks for filing.

**Comment by thebreiflabb** [comment]
@rozele It works great with 0.38.1, thank you very much.
